python/tvm/runtime/__init__.py

```diff
@@ -40,3 +40,4 @@
 )
 
 from . import executor
+from . import disco
```

Comment:
Not sure if it's safe when runtime is not compiled with disco.

Reply:
Hmm, good point.  The `tvm._ffi.register_object("runtime.disco.*")` calls would raise an error if disco isn't available.

This was primarily to ensure that all packaged funcs generated by `tvm.runtime.disco` were available within the disco sub-process.  I'll try a build without disco enabled, and see what is required to make sure that the module import works either way.